src/wasm/wasm-io.cpp

```diff
@@ -24,19 +24,31 @@
 // binary.
 //
 
-#include "wasm-io.h"
 #include "support/debug.h"
 #include "wasm-binary.h"
 #include "wasm-s-parser.h"
+#include "wat-parser.h"
+
+#include "wasm-io.h"
```

Comment:
I don't think I've ever noticed that clang-format puts the corresponding header at the top before, and I would have expected it to put that header at the end, if anything. But I guess it would be best to just do what clang-format wants. I'll remove the unnecessary spacing and fix the order here.